### PR TITLE
[BEAM-8003] Remove Perfkit leftovers

### PR DESCRIPTION
**Pull request by lgajowy**
Removes all mentions of PKB in documentation that are out of date. Removes Gradle task that is obsolete since we resigned from using Perfkit. Removes Perfkit word from file-based tests description (makes no sense anymore).

@markflyhigh could you take a look?

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)
XLang | --- | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | --- | ---

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
Run Java TextIO Performance Test HDFS

**Comment by lgajowy** [comment]
@markflyhigh pinging kindly. :) Let me know if you have no time to review and I will reassign.

**Comment by lgajowy** [comment]
Reassigning: @mwalenia could you take a look?

**Comment by lgajowy** [comment]
@aromanenko-dev if you have time could you take a look too?

**Comment by aromanenko-dev** [comment]
Run Website_Stage_GCS PreCommit

File: website/src/documentation/io/testing.md [hunk 1]
```diff
@@ -266,10 +169,9 @@ If you're using Kubernetes scripts to host data stores, make sure you can connec
     1.  JDBC: `kubectl delete -f .test-infra/kubernetes/postgres/postgres.yml`
     1.  Elasticsearch: `bash .test-infra/kubernetes/elasticsearch/teardown.sh`
 
-##### integrationTest Task {#integration-test-task}
-
-Since `performanceTest` task involved running PerfkitBenchmarker, we can't use it to run the tests manually. For such purposes a more "low-level" task called `integrationTest` was introduced.  
+#### Running the test {#running-the-test}
```
Comment (markflyhigh) [inline website/src/documentation/io/testing.md]: The section here and below `Running Integration Tests on Pull Requests {#running-on-pull-requests}` seems two ways of running the tests. Should they be in same level? Also this seems focus on running tests with local Gradle, but not very clear to me.

Comment (lgajowy) [reply website/src/documentation/io/testing.md]: Thanks @markflyhigh! I don't think they should. This paragraph describes the way to run a particular test alone. It's a part of a more highlevel paragraph about running any test on your machine.  The other describes the ability of phrase triggering using Beam's CI in general.

I added some sentences to make this more clear. Let me know if this was helpful. :)

File: website/src/documentation/io/testing.md [hunk 2]
```diff
@@ -266,10 +169,9 @@ If you're using Kubernetes scripts to host data stores, make sure you can connec
...
 
+integrationTest is a dedicated gradle task for running IO integration tests.
```
[the remark this answers is not in the document]
Comment (lgajowy) [reply website/src/documentation/io/testing.md]: 👍

**Comment by lgajowy** [comment]
@markflyhigh @aromanenko-dev could you take another look?

**Comment by tvalentyn** [comment]
We also have Perfkit leftovers in https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_PerformanceTests_Python.groovy - do we have plan to move those benchmarks off of PKB?

**Comment by aromanenko-dev** [comment]
Run Python PreCommit

**Comment by aromanenko-dev** [comment]
Run JavaBeamZetaSQL PreCommit

**Comment by aromanenko-dev** [comment]
Run Portable_Python PreCommit

**Comment by aromanenko-dev** [comment]
Run Portable_Python PreCommit

**Comment by aromanenko-dev** [comment]
LGTM

**Comment by lgajowy** [comment]
Thank you all! :)

**Pull request merged**